cli.js

```diff
@@ -61,4 +57,8 @@ for (var i = 0; i < process.argv.length; i++) {
 delete options.tasks;
 options.cwd = cwd;
 
-gulp.run(tasks, options);
+gulp.run(tasks, options, function(error) {
+    if (error) {
+        process.exitCode = 1;
```

Comment:
Node.js docs say it's safer to set the exitCode rather than call process.exit, as that might abruptly terminate async processes that are still cleaning up. I could be convinced that it's okay to do an explicit exit just to make sure nothing hangs. WDYT @tfennelly ?